mmseg/models/backbones/fast_scnn.py

```diff
@@ -290,6 +309,8 @@ class FastSCNN(BaseModule):
             dict(type='ReLU')
         align_corners (bool): align_corners argument of F.interpolate.
             Default: False
+        dw_act_cfg (dict): Activation config of depthwise ConvModule. If it is
```

Comment:
Same

Reply:
`'default'` is a string value at `DepthwiseSeparableConvModule`. In `LearningToDownsample` module, I want to set it to `None`.